src/graphene-rect.h

```diff
@@ -172,6 +177,9 @@ void                    graphene_rect_expand            (const graphene_rect_t
                                                          const graphene_point_t *p,
                                                          graphene_rect_t        *res);
 
+GRAPHENE_AVAILABLE_IN_1_10
+void                    graphene_rect_print             (const graphene_rect_t *r);
```

Comment:
I'm not overly fond of these; I'm already regretting having added `graphene_matrix_print()`.

Reply:
So, want me drop this commit? :(

I mean, I understand it's just a dev-facility, but all the times I've to debug stuff I hate to loose precious seconds in doing all the `%d` repetitions around :-P

Reply:
Yes, I really don't want this in the public API.